src/test/java/eu/miaplatform/service/HTTPServiceClientTest.java

```diff
@@ -0,0 +1,115 @@
+package eu.miaplatform.service;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import eu.miaplatform.service.*;
+import okhttp3.Response;
+import org.junit.*;
+
+import java.io.IOException;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class HTTPServiceClientTest {
```

Comment:
This class testes service class not the httpserviceclient interface, lets name it accordingly

Reply:
Done